src/xrScriptEngine/script_engine.cpp

```diff
@@ -524,7 +524,7 @@ struct raii_guard : private Noncopyable
 #ifdef DEBUG
             static const bool break_on_assert = !!strstr(Core.Params, "-break_on_assert");
 #else
-            static const bool break_on_assert = true;
+            static const bool break_on_assert = false;	// xxx: true will hide real lua error, LuaJIT problem?
```

Comment:
With my changes, current log looks like this:

```
! SCRIPT ERROR: see call_stack for details!
 
FATAL ERROR
 
[error] Expression    : <no expression>
[error] Function      : CScriptEngine::lua_pcall_failed
[error] File          : script_engine.cpp
[error] Line          : 767
[error] Description   : fatal error
[error] Arguments     : LUA error: No matching overload found, candidates:
void ShowDialog(CUIDialogWnd&,bool)
Passed arguments [2]: userdata (CUIMessageBoxEx&), nil (nil)
 
stack traceback:
	[C]: in function 'ShowDialog'
	...r - call of pripyat\gamedata\scripts\ui_main_menu.script:140: in function <...r - call of pripyat\gamedata\scripts\ui_main_menu.script:138>
 
```

Game stiil break on assert, just in another place.